### PR TITLE
* Fixing really bad issue with -it arg...

### PR DESCRIPTION
**Pull request by Shoh**
Short Description: Fixing really ugly issues and some minor mistakes/typos

Fixes:
- Changing default max_steps to 5
- Fixed documentation where necessary
- Re-adding pokeball count back to output

### CODE REVIEW

**Comment by solderzzc** [comment]
Let me read code here.

File: pokecli.py [hunk 1]
```diff
@@ -97,9 +97,9 @@ def init_config():
         "-it",
         "--initial_transfer",
         help=
-        "Transfer all pokemon with same ID on bot start, except pokemon with highest CP. Can receive a CP number to not transfer above it",
+        "Transfer all duplicate pokemon with same ID on bot start, except pokemon with highest CP. Accepts a number to prevent transferring pokemon with a CP above the provided value.  Default is 0 (aka transfer none).",
         type=int,
-        default=999999)
+        default=0)
```
Comment (solderzzc) [inline pokecli.py]: What will happen set default to 0?

Comment (solderzzc) [inline pokecli.py]: oh, it was transfer all !!

**Comment by Shoh** [comment]
Of course.  Please do. :)  It's mostly fixing typos and re-adding some previously overwritten changes.

File: pokecli.py (hunk 1 above)
Comment (Shoh) [inline pokecli.py]: Yeah, setting it to 0 disables it by default.

**Pull request merged**